### PR TITLE
Add a feature of merge isomers

### PR DESCRIPTION
**Pull request by yohsama**
Add a feature of merge isomers
-----------------
In the case of MD with long carbon chain molecules, especially the carbonyne chains, the long carbon chain molecules will be identified as a variety of different isomers, a large number of molecules were identified, and each one has a weak signal and is easily erased in hmm.
 e.g.
> [H]C=C=C(Cl)C=C=CCCl,
> [H]C=C=C(Cl)C=CC=CCl,
> [H]C=C=C(Cl)C#CCCCCl
> ...

the feature is for merging those isomers and using the highest frequency one as the representative. 

A new argument --miso
0, off 
1, merge the isomers with same atoms and same bond-network but different bond levels;
 
e.g.
>  [H]C=C=C(Cl)C=C=CCCl,
>  [H]C=C=C(Cl)C=CC=CCl, 
>  will be consider as one isomer

2, merge the isomers with same atoms with different bond-network.

e.g.
> [H]C(Cl)=C=CC=C=CCCl,
> [H]C=C=C(Cl)C=CC=CCl, 
> will be consider as one isomer

-------------
Hope this is useful, XD. If there is a better way, please share it with me.
Sorry my English sucks, hahaha. XD

### CODE REVIEW

File: reacnetgenerator/_mergeiso.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    def __init__(self, rng):
+        SharedRNGData.__init__(self, rng, ['miso', 'moleculetempfilename'],['temp1it'])
+
+    def merge(self):
+        if self.miso > 0:
+            self._mergeISO()
+        self.returnkeys()
+
+    def _mergeISO(self):
+        with open(self.moleculetempfilename, 'rb') as ft,open('/data/home/cx/test/reac/test.dat', 'wb') as ft2:
```
Comment (njzjz) [inline reacnetgenerator/_mergeiso.py:17]: ft2 is for debugging? The path is not correct.

**Review by njzjz — CHANGES_REQUESTED** [review]

File: reacnetgenerator/_mergeiso.py (hunk 1 above)
Comment (yohsama) [reply reacnetgenerator/_mergeiso.py:17]: yes, I forget to remove this.

**Review by yohsama — COMMENTED** [review]
remove code for debug

**Review by njzjz — APPROVED** [review]

**Pull request merged**